pkg/cvo/updatepayload.go

```diff
@@ -248,12 +255,6 @@ func (r *payloadRetriever) pruneJobs(ctx context.Context, retain int) error {
 		// Ignore jobs not beginning with operatorName
 		case !strings.HasPrefix(job.Name, r.operatorName+"-"):
 			break
```

Comment:
Why is it that we no longer have to ignore jobs that have not yet started or are still active? Will these never show up now because they will have been pruned?

Reply:
when we are getting ready to launch the new job, we have already given up on the old job, so we are pruning those, even if they were still pending or running.  The new job will succeed, or not, and will remain around until the CVO decides to launch a successor.